trymito.io/components/CTAButtons/CTAButtons.tsx

```diff
@@ -19,13 +19,13 @@ const CTAButtons = (props: {variant: 'download' | 'contact'}): JSX.Element => {
                     href="mailto:founders@sagacollab.com"
                 />
             }
-            <p className={styles.cta_subbutton}>
+            <h3 className={styles.cta_subbutton}>
```

Comment:
This is kinda ugly IMO 
<img width="300" alt="Screenshot 2023-01-27 at 5 16 01 PM" src="https://user-images.githubusercontent.com/18709905/215216097-5e0e5e6e-ae1b-4583-8373-1d82c09e4dc8.png">